File: tests/fixtures/pure-py-typed/src/session.py
import random
import string
from .types import Session
from .cache import Cache
from .constants import SESSION_TTL_S

class SessionStore:
    def __init__(self, cache: Cache) -> None:
        self._cache = cache

    def create(self, user_id: str) -> Session:
        token = ''.join(random.choices(string.ascii_lowercase, k=32))
        return Session(token=token, user_id=user_id)

File: tests/fixtures/pure-ts/src/session.ts
import type { Session } from './types.js'
import { Cache } from './cache.js'
import { SESSION_TTL_MS } from './constants.js'
export class SessionStore {
  constructor(private cache: Cache) {}
  create(userId: string): Session {
    return { token: Math.random().toString(36), userId, expiresAt: new Date(Date.now() + SESSION_TTL_MS) }
  }
}
